#  boolean to state, if the robot is moving
bool robot_moving


# aruco coordinates in the robot coordinate system
geometry_msgs/Point aruco_corner_0
geometry_msgs/Point aruco_corner_1
geometry_msgs/Point aruco_corner_2
geometry_msgs/Point aruco_corner_3